src/test/java/org/broadinstitute/hellbender/tools/picard/sam/FilterReadsIntegrationTest.java

```diff
@@ -112,8 +112,7 @@ public void testReadFilter(
         }
 
         if (!writeReadsFile) { // defaults to true
```

Comment:
Something bogus here - the condition says `if not writeReadsFile then use --WRITE_READS_FILES`